components/profile/activityTab/History.vue

```diff
@@ -10,7 +10,7 @@
 
     <div v-if="!desktop" class="is-flex is-justify-content-flex-end my-5">
       <Pagination
-        v-model="currentPage"
+        :value="currentPage"
```

Comment:
looks like root issue is in `Pagination` component which don't emit updated value